experiment-train/README.md

```diff
@@ -0,0 +1,48 @@
+### Cortex Experiment Train SKill
+
+Cortex Skill to train save model in Experiment
```

Comment:
More words /description of what is going on why use this ... what it does ?

Reply:
Agreed.  Need a paragraph on why would one use this Skill. Also, any pre-reqs to be documented.